psf-new-code-of-conduct.md

```diff
@@ -44,16 +47,16 @@ Examples of unacceptable behavior by participants include:
 
 Community members asked to stop any inappropriate behavior are expected to comply immediately.
 
-### Reversisms
+### Communicating About Assumptions and Boundaries
 
-The Python community prioritizes marginalized people's safety over privileged people's comfort. The Python Code of Conduct team will not act on complaints regarding:
+The Python Community welcomes people who have traditionally been marginalized within our community: we allow them the space to discuss behaviors, stereotypes, and assumptions that make them feel unwelcome. We also believe that helping others understand how various communications are received can be constructive.
```

Comment:
@chrisjrn and I talked this one over a bit and I have to 👍 this a lot as as @chrisjrn pointed out, not only does this give us language to scope to our community more directly but provides us with why this is important.